packages/cozy-intent/src/view/components/WebviewIntentProvider.spec.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+import { render, screen } from '@testing-library/react'
+import 'mutationobserver-shim'
```

Comment:
why?

maybe we should mock more instead of installing devDependencies that are not part of the business of cozy-intent.

Reply:
I forgot to change the PR target after a rebase. This comment is related to : https://github.com/cozy/cozy-libs/pull/1438